luma/core/serial.py

```diff
@@ -2,22 +2,31 @@
 # Copyright (c) 2017 Richard Hull and contributors
 # See LICENSE.rst for details.
 
+"""
+Encapsulates sending commands and data over a serial interface, whether that
+is I²C, SPI or bitbanging GPIO.
+"""
+
 import errno
 
 import luma.core.error
 
 
 class i2c(object):
     """
-    Wrap an `I2C <https://en.wikipedia.org/wiki/I%C2%B2C>`_ interface to
-    provide data and command methods.
-
-    :param bus: I2C bus instance.
+    Wrap an `I²C <https://en.wikipedia.org/wiki/I%C2%B2C>`_ (Inter-Integrated
+    Circuit) interface to provide :py:func:`data` and :py:func:`command` methods.
```

Comment:
I have a suspicion it's these unicode things, e.g. I²C, is screwing up the intersphinx build.